### ISSUE TITLE
Function.extractParams is deprecated.

### ISSUE THREAD

**Issue by kankaristo** [ohanhi/atom-highlight-bad-chars#7]
Assigning custom properties to a marker when creating/copying it is
deprecated. Please, consider storing the custom properties you need in
some other object in your package, keyed by the marker's id property.
```
Function.extractParams (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/marker.js:43:14)
MarkerLayer.markRange (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/marker-layer.js:196:71)
DisplayMarkerLayer.markBufferRange (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:90:62)
TextEditor.markBufferRange (/usr/share/atom-beta/resources/app.asar/src/text-editor.js:1753:38)
<unknown> (/home/sami/.atom/packages/highlight-bad-chars/lib/highlight-bad-chars.coffee:26:29)
<unknown> (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:950:18)
```

**Comment by WaKeMaTTa** [comment]
Please fix this.

**Comment by ohanhi** [comment]
I would if I knew how. There's nothing on the Atom plugin documentation on this. Why don't you give it a try and submit a PR?

**Comment by WaKeMaTTa** [comment]
@ohanhi Check this PR https://github.com/smashwilson/merge-conflicts/pull/265

**Comment by WaKeMaTTa** [comment]
https://github.com/ohanhi/atom-highlight-bad-chars/blob/f5d34fa49f28e616a58abe6bb1efd34bc114b8c0/lib/highlight-bad-chars.coffee#L84

**Comment by WaKeMaTTa** [comment]
I have done a PR https://github.com/ohanhi/atom-highlight-bad-chars/pull/13

**Comment by ohanhi** [comment]
Thanks! Published as 1.0.7
